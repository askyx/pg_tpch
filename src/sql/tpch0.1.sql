select * from dbgen(0.1);


-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
