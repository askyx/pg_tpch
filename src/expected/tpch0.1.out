select * from dbgen(0.1);
   tab    | row_count 
----------+-----------
 region   |         5
 nation   |        25
 supplier |      1000
 customer |     15000
 part     |     20000
 partsupp |     80000
 orders   |    150000
 lineitem |    600572
(8 rows)

-- select * from tpch;
-- select * from tpch;
-- select * from tpch(1,1,1);
-- select * from tpch(2,2,2);
